iosHackthonApp/Modules/Features/AddMeal/Sources/AddMeal/AddMealView.swift

```diff
@@ -143,8 +91,144 @@ public struct AddMealView: View {
     }
 }
 
+// MARK:- Views
+extension AddMealView {
```

Comment:
Very minor but if you add `private` here you don't have to add it to the first-level nested types here because they're automatically private

Reply:
Yeah sure :-) I'll do the same for the functions extension too 👍